### DISCUSSION TITLE
Slurm-web lodding with Errros

### DISCUSSION THREAD

**Discussion by rahulp5428** [rackslab/Slurm-web#245] [Q&A]
![image](https://github.com/rackslab/slurm-web/assets/157102198/da3ad9fc-0c12-44fc-829e-2022d978e987)

Please help me with i try #198  setup in docker containers.

**Comment by rezib** [comment]
Hi @rahulp5428,

Thank for your interest in Slurm-web. Your issue is more a community support request than an actual bug. In this project, the Q&A section of project discussions are preferred over issues for such kind of requests. For this reason, I'll convert this issue into a discussion.

Best regards,

**Comment by rezib** [comment]
This question was about deprecated Slurm-web 2, then I close the discussion.
